src/main/kotlin/no/nav/familie/ef/sak/ekstern/bisys/BisysBarnetilsynService.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.familie.ef.sak.ekstern.bisys
+
+import no.nav.familie.ef.sak.barn.BarnService
+import no.nav.familie.ef.sak.fagsak.FagsakService
+import no.nav.familie.ef.sak.fagsak.domain.Fagsak
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.PersonService
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl.identer
+import no.nav.familie.ef.sak.tilkjentytelse.TilkjentYtelseService
+import no.nav.familie.ef.sak.vedtak.historikk.EndringType
+import no.nav.familie.eksterne.kontrakter.bisys.BarnetilsynBisysPeriode
+import no.nav.familie.eksterne.kontrakter.bisys.BarnetilsynBisysResponse
+import no.nav.familie.eksterne.kontrakter.bisys.Datakilde
+import no.nav.familie.eksterne.kontrakter.bisys.Periode
+import no.nav.familie.kontrakter.felles.ef.StønadType
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class BisysBarnetilsynService(
+        private val personService: PersonService,
+        private val fagsakService: FagsakService,
+        private val barnService: BarnService,
+        private val tilkjentYtelseService: TilkjentYtelseService) {
+
+    fun hentPerioderBarnetilsyn(personIdent: String, fomDato: LocalDate): BarnetilsynBisysResponse {
+
+        val personIdenter = personService.hentPersonIdenter(personIdent).identer()
+        val fagsak: Fagsak = fagsakService.finnFagsak(personIdenter, StønadType.BARNETILSYN)
+                             ?: error("Kunne ikke finne fagsak for personident")
+
+        val historikk = tilkjentYtelseService.hentHistorikk(fagsak.id, null)
+                .filter { it.endring?.type != EndringType.FJERNET }
+                .filter { it.endring?.type != EndringType.ERSTATTET }
+                .filter { it.andel.beløp > 0 && it.andel.stønadFra >= fomDato }
```

Comment:
`it.andel.stønadFra >= fomDato` er det riktig? 
Hva hvis vi har en periode
01.2021 - 12.2021 og de henter med dato 06.2021? Hva er ønskelig å returnere då?

Reply:
Ja, sant, den må ligge mellom. La til mindre eller lik f.o.m dato, og større eller lik t.o.m dato